recipes/smoothxg/build.sh

```
@@ -1,15 +1,28 @@
 export CXX_INCLUDE_PATH=${PREFIX}/include
-sed -i 's/-msse4.1/-march=sandybridge -Ofast/g' deps/spoa/CMakeLists.txt
-sed -i 's/-march=native/-march=sandybridge -Ofast/g' deps/spoa/CMakeLists.txt
-sed -i 's/-march=native/-march=sandybridge -Ofast/g' deps/abPOA/CMakeLists.txt
-cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Generic -DEXTRA_FLAGS='-march=sandybridge -Ofast'
-cmake --build build --verbose
+
+case $(uname -m) in
+    aarch64 | arm64)
+        ARCH_FLAGS=""
+        sed -i 's/-msse4.1/-Ofast/g' deps/spoa/CMakeLists.txt
+    *)
```

Shellcheck

[error]  Fix any mentioned problems and try again.

(SC1072)